src/js/components/stock-movement/outbound/StockMovementOutboundTable.jsx

```diff
@@ -42,20 +48,79 @@ const StockMovementOutboundTable = ({
     exportStockMovements,
     exportPendingShipmentItems,
     deleteConfirmAlert,
+    approveRequest,
+    rejectRequest,
+    rollbackRequest,
   } = useOutboundListTableData(filterParams);
 
   const getStatusTooltip = status => translate(
     `react.stockMovement.status.${status.toLowerCase()}.description.label`,
     status.toLowerCase(),
   );
 
+  const approverActions = useCallback(
+    (row) => {
+      const {
+        statusCode,
+        identifier,
+      } = row.original;
+      const actions = [];
+
+      if (statusCode === RequisitionStatus.PENDING_APPROVAL) {
+        const approveAction = {
+          defaultLabel: 'Approve',
+          label: 'react.stockMovement.action.approve.label',
+          leftIcon: <RiCheckFill />,
+          onClick: id => approveRequest(id, identifier),
+        };
+        actions.push(approveAction);
+      }
+      if (statusCode === RequisitionStatus.PENDING_APPROVAL) {
```

Comment:
you can move the content of that if into the if above, since those are the same "ifs"